daemon/protocol/python/tests/daemon_cli_test.py

```diff
@@ -5,140 +5,180 @@
 import shutil
 import subprocess
 import tempfile
-import time
-import unittest
 
 
+gopath = re.split(":", os.environ['GOPATH'])[0]
```

Comment:
removed the unittest nature from the test - make will not run in Travis if it is a unittest (though make runs locally)